### DISCUSSION TITLE
Vue2.7 official project template

### DISCUSSION THREAD

**Discussion by weijian-zhu** [vuejs/vue-cli#7240] [General]
Has anyone tried a project to fully configure Vue2.7? This includes syntax that works with vue2 and vue3, correct ts syntax highlighting, correct eslint tooltips for vue2.7, etc. I am having trouble with this, syntax highlighting is not configured very well or even wrong, if only there was an official template for reference

**Comment by tony19** [comment]
Make sure you're using [Volar](https://marketplace.visualstudio.com/items?itemName=vue.volar), not Vetur.

**Reply by weijian-zhu** [reply]
I did use Volar

**Reply by weijian-zhu** [reply]
For example, when I define component metadata in vue2.7, ts will report an error. I wonder what is wrong with my configuration
![image](https://user-images.githubusercontent.com/50317466/179684812-6c8536a0-41fb-49ac-99fb-93d45a4c0aa6.png)
